src/igvfd/audit/measurement_set.py

```diff
@@ -88,9 +88,9 @@ def audit_unspecified_protocol(value, system):
 @audit_checker('MeasurementSet', frame='object')
 def audit_inconsistent_readout(value, system):
     '''
-        audit_detail: CRISPR-based and MPRA assays are required to specify a readout, other assays should not include one.
+        audit_detail: CRISPR-based and MPRA assays are required to specify a readout, other assays should not include one. Also readout is not expected be the same as assay term.
```

Comment:
instead of: "audit_detail: CRISPR-based and MPRA assays are required to specify a readout, other assays should not include one. Also readout is not expected be the same as assay term."

 "audit_detail: CRISPR-based and MPRA assays are required to specify a readout that is different from the assay term of the measurement set, other assays should not include readout specification."

Reply:
Fixed.